CHANGELOG.md

```diff
@@ -5,7 +5,6 @@
 
 **Implemented enhancements:**
 
-- Do not allow hyphenated cookbook names to be generated [\#915](https://github.com/chef/chef-dk/pull/915)
```

Comment:
when we do a release, we "auto" generate the changelog and releasenotes based on GH commits. So if this were merged, it would show up in 0.17.